core/api/transport/impl/http/http_session.cpp

```diff
@@ -133,4 +136,8 @@ namespace kagome::api {
   void HttpSession::post(std::function<void()> cb) {
     boost::asio::post(strand_, std::move(cb));
   }
+
+  bool HttpSession::allowUnsafe() const {
```

Comment:
The name sounds as if it made unsafe allowed instead of returning whether it is allowed. isUnsafeAllowed or allowsUnsafe would be clearer.